packages/core/src/components/templates/index.js

```diff
@@ -17,6 +17,7 @@
  */
 
 export {default as AttachedFilesView} from './AttachedFilesView/AttachedFilesView';
+export {default as useLoaderListner} from './Loader/use-loader-listener';
```

Comment:
Add an index file in Loader folder to export this file and the next ones

Reply:
I did that in the following ticket